src/twisted/test/test_defer.py

```diff
@@ -176,6 +176,44 @@ def testTwoCallbacks(self):
         self.assertEqual(self.callbackResults, (("hello",), {}))
         self.assertEqual(self.callback2Results, (("hello",), {}))
 
+    def testAddCallbacksNoneErrback(self):
```

Comment:
Nitpick: new test names should be `test_wordsWordsWords`. Not gonna block on that though ;-)

Reply:
Fine by me. DeferredTests doesn't… ok it does sometimes.  I was doing "when in Rome". Fixing.

Reply:
Thanks!  And yeah, `DeferredTests` is *super* old.